pkg/cmd/server/start/start_master.go

```diff
@@ -451,7 +451,7 @@ func StartAPI(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) error {
 
 	var standaloneAssetConfig *origin.AssetConfig
 	if oc.WebConsoleEnabled() {
-		config, err := origin.BuildAssetConfig(*oc.Options.AssetConfig)
+		config, err := origin.NewAssetConfig(*oc.Options.AssetConfig, oc.Options.KubernetesMasterConfig.AdmissionConfig.PluginConfig)
```

Comment:
I thought it not unlikely that perhaps another admission controller might also have config that goes into the web console asset bundle. But *shrug* it is not too hard to just make that change when is needed. I can pass just the config for the one plugin we care about now.